src/main/java/picard/fingerprint/CrosscheckReadGroupFingerprints.java

```diff
@@ -86,6 +91,8 @@
     @Override
     protected int doWork() {
 
+        if (EXPECT_ALL_READ_GROUPS_TO_MATCH) EXPECT_ALL_GROUPS_TO_MATCH = EXPECT_ALL_READ_GROUPS_TO_MATCH;
```

Comment:
Since this is java it should have `{` `}` around the body.

All the other options that migrated to the base class kept their original names. Was the original name incorrect? Or is it now no longer specific to read groups?